index-handlers/src/main/java/no/sikt/nva/nvi/index/xlsx/ExcelWorkbookGenerator.java

```diff
@@ -47,6 +47,15 @@ public boolean equals(Object o) {
         return Objects.equals(headers, that.headers) && Objects.equals(data, that.data);
     }
 
+    @Override
+    @JacocoGenerated
+    public String toString() {
+        return "ExcelWorkbookGenerator{"
+               + "headers=" + headers
+               + ", data=" + data
+               + '}';
+    }
+
```

Comment:
It seems like you can simply add `implements JsonSerializable` instead of implementing hardcoded toString() method, or:

    public String toString() {
        return this.toJsonString();
    }

Reply:
I didn't find json as a suitable way to represent tabular data, see `toString`-refactor in latest commits